Add forem meta tags (#10747)
forem/forem: docs/technical-overview/compatibility.md

# Compatibility

> To go along with effective administration and putting user security and wellbeing first, for a forem to be most useful, it maximizes its technical compatibility.

Reviewer: ```suggestion
To go along with effective administration and putting user security and wellbeing first, for a Forem to be most useful, it maximizes its technical compatibility.
```

Reviewer 2: Agree with @jacobherrington's suggestions here and also want to add that we should use `well-being`, the standard noun format, rather than `wellbeing` here :)

> To go along with effective administration and putting user security and wellbeing first, for a Forem to be most useful, it maximizes its technical compatibility.

Author: ```suggestion
To go along with effective administration and putting user security and well-being first, for a Forem to be most useful, it maximizes its technical compatibility.
```

Compatability means:

- Staying updated. We need to maintain overall compatability by keeping the ecosystem close to the latest version as proactively as we can.

> - Maintaining all programmatic interfaces, whether they be the [API](https://api.forem.com) or forem meta tags which can be consumed for forem validation.

Reviewer: ```suggestion
- Maintaining all programmatic interfaces, whether they be the [API](https://api.forem.com) or Forem meta tags which can be consumed for Forem validation.
```

- Maintaining human interfaces which allow users to find where they need to go and conduct all necessary customizations.

> - Maintain great performance such that the web-based forem ecosystem can provide a great experience in all network conditions and in comparison to more closed ecosystems.

Reviewer: ```suggestion
- Maintaining great performance such that the web-based Forem ecosystem can provide a great experience in all network conditions and in comparison to more closed ecosystems.
```
